modules/atomic-widgets/widgets/atomic-heading.php

```diff
@@ -32,8 +32,13 @@ protected function render() {
 		$escaped_tag = Utils::validate_html_tag( $settings['tag'] );
 		$escaped_title = esc_html( $settings['title'] );
 
+		$class = '';
+		if ( ! empty( $settings['classes'] ) ) {
+			$class = "class='" . $settings['classes'] . "'";
```

Comment:
```suggestion
			$class = "class='" . esc_attr( $settings['classes'] ) . "'";
```

for safety measures 🤷